### PR TITLE
Replace GetGridUid() with transformSystem.GetGrid()

### PR DESCRIPTION
**Pull request by Plykiya**
## About the PR

Title

## Why / Balance

Project 0 warnings

## Technical details

Pretty uneventful drop-in replace

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

beep boop

### CODE REVIEW

**Comment by Plykiya** [comment]
suffering

**Comment by Plykiya** [comment]
image
wait a minute, all the tests passed locally

**Comment by Plykiya** [comment]
image

strange, the tests all pass locally.

**Comment by Winkarst-cpu** [comment]
ubuntu-latest is broken.

**Comment by Plykiya** [comment]
image
image

maaan, I don't know

**Comment by Winkarst-cpu** [comment]
Just say that failed check is not related to the PR.

**Comment by Plykiya** [comment]
> Just say that failed check is not related to the PR.

image

yeah alright

**Comment by Plykiya** [comment]
I'll just leave those two random test fails for someone else to deal with in another PR 👍

**Comment by Winkarst-cpu** [comment]
> I'll just leave those two random test fails from PathfindingSystem.Grid and PuddleSystem for someone else to deal with in another PR 👍

All checks have passed 😄

File: Content.Server/Administration/Commands/WarpCommand.cs [hunk 1]
```diff
@@ -63,8 +63,8 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                         // Sort so that warp points on the same grid/map are first.
                         // So if you have two maps loaded with the same warp points,
                         // it will prefer the warp points on the map you're currently on.
-                        var aGrid = a.GetGridUid(_entManager);
-                        var bGrid = b.GetGridUid(_entManager);
+                        var aGrid = _entManager.System<SharedTransformSystem>().GetGrid(a);
+                        var bGrid = _entManager.System<SharedTransformSystem>().GetGrid(b);
```
Comment (metalgearsloth) [inline Content.Server/Administration/Commands/WarpCommand.cs:68]: Re-use system plz.

File: Content.Server/Administration/Commands/WarpCommand.cs [hunk 2]
```diff
@@ -118,7 +118,7 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 }
 
                 var xform = _entManager.GetComponent<TransformComponent>(playerEntity);
-                xform.Coordinates = coords;
+                _entManager.System<SharedTransformSystem>().SetCoordinates(playerEntity, coords);
                 xform.AttachToGridOrMap();
```
Comment (metalgearsloth) [inline Content.Server/Administration/Commands/WarpCommand.cs:122]: Same here also attachtogridormap also bad.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Pull request closed without merging**